src/app/pages/charging-stations/charging-station-start-transaction/charging-stations-start-transaction-dialog-component.ts

```diff
@@ -57,8 +58,9 @@ export class ChargingStationsStartTransactionDialogComponent implements OnInit {
     // Set
     this.title = data.title;
     this.chargeBoxID = data.chargeBoxID;
+    this.siteID = data.siteID;
     this.loggedUser = centralServerService.getLoggedUser();
-    this.canListUsers = this.authorizationService.canListUsers();
+    this.canListUsers = this.authorizationService.isSiteAdmin(this.siteID);
```

Comment:
This should be sent back by the backend when retrieving the default tag/car...

Reply:
Done